src/Security/Member.php

```diff
@@ -999,7 +991,7 @@ public function addToGroupByCode($groupcode, $title = "")
      */
     public function removeFromGroupByCode($groupcode)
     {
-        $group = Group::get()->filter(['Code' => $groupcode])->first();
+        $group = Group::get()->find('Code', $groupcode);
```

Comment:
Not directly related but just tidying this up while I'm here. The end result identical.